arion-pkgs.nix

```
@@ -1 +1,15 @@
-import <nixpkgs> { system = "x86_64-linux"; }
+let flake = (import
+  (
+    let lock = builtins.fromJSON (builtins.readFile ./flake.lock); in
+    fetchTarball {
+      url = "https://github.com/edolstra/flake-compat/archive/${lock.nodes.flake-compat.locked.rev}.tar.gz";
+      sha256 = lock.nodes.flake-compat.locked.narHash;
+    }
+  )
+  { src = ./.; }
+);
+in
+import (flake.inputs.nixpkgs) {
+  systeem = "x86-64_linux";
+  overlays = [ flake.overlay ];
+}
```

I needed to change this to:

```suggestion
import (flake.defaultNix.inputs.nixpkgs) {
  system = "x86_64-linux";
  overlays = [ flake.defaultNix.overlays.default ];
}
```

Also does using `x86_64-linux` mean someone not on linux will have to change this line to run the arion stuff? If so I'll put it in the documentation